src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/IndexStateManagementPlugin.kt

```diff
@@ -15,10 +15,55 @@
 
 package com.amazon.opendistroforelasticsearch.indexstatemanagement
 
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.models.ManagedIndex
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.JobSchedulerExtension
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.ScheduledJobParameter
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.ScheduledJobParser
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.ScheduledJobRunner
+import org.apache.logging.log4j.LogManager
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParserUtils
 import org.elasticsearch.plugins.Plugin
 
-internal class IndexStateManagementPlugin: Plugin() {
+internal class IndexStateManagementPlugin : JobSchedulerExtension, Plugin() {
+
+    private val logger = LogManager.getLogger(javaClass)
+
     companion object {
         const val PLUGIN_NAME = "opendistro-ism"
+        const val POLICY_BASE_URI = "/_opendistro/_ism/policies"
```

Comment:
do we have a convention about the REST uri? such as when to use "_"?

Reply:
Yes, following conventions from [Alerting](https://github.com/opendistro-for-elasticsearch/alerting/blob/56c49d9fe8c8332be88a9295298ded8ae5d35172/alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/AlertingPlugin.kt#L83)